### DISCUSSION TITLE
[Q] I'm sorry.excuse me.Is there a better way ?

### DISCUSSION THREAD

**Discussion by Soneqt** [exceljs/exceljs#2367] [Q&A]
I'm sorry.excuse me.Is there a better way ?
If so.I have to write it a lot of times

 const _sheet1 = _workbook.addWorksheet("sheet1");
 _sheet1.getCell("B2").dataValidation = {
                type: "list",
                allowBlank: true,
                formulae: ['"A,B,C,D,E,F"'],
            };
 _sheet1.getCell("B3").dataValidation = {
                type: "list",
                allowBlank: true,
                formulae: ['"A,B,C,D,E,F"'],
            };
 _sheet1.getCell("B4").dataValidation = {
                type: "list",
                allowBlank: true,
                formulae: ['"A,B,C,D,E,F"'],
            };
...........
..........
..........
 _sheet1.getCell("B100").dataValidation = {
                type: "list",
                allowBlank: true,
                formulae: ['"A,B,C,D,E,F"'],
            };

**Comment by Turtlefight** [comment]
You can use a simple for-loop for this:
```javascript
for(let i = 2; i <= 100; i++) {
  worksheet.getCell(i, 2).dataValidation = {
    type: "list",
    allowBlank: true,
    formulae: ['"A,B,C,D,E,F"']
  };
}
```

**Comment by Soneqt** [comment]
thank you.
Is there no built-in method?
